base/applications/games/solitaire/lang/fr-FR.rc

```diff
@@ -16,15 +16,15 @@ CAPTION "Options"
 FONT 8, "MS Shell Dlg"
 BEGIN
     GROUPBOX "Cartes", -1, 7, 7, 90, 40
-    AUTORADIOBUTTON "En tirer une", IDC_OPT_DRAWONE, 14, 19, 70, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "En tirer trois", IDC_OPT_DRAWTHREE, 14, 32, 70, 10
-    GROUPBOX "Scoring", -1, 100, 7, 75, 53
+    AUTORADIOBUTTON "Tirer une carte", IDC_OPT_DRAWONE, 14, 19, 70, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "Tirer trois cartes", IDC_OPT_DRAWTHREE, 14, 32, 70, 10
```

Comment:
No idea what Windows does, but I personally tend to prefer the previous translations.

Reply:
As you prefer. I reverted.